src/components/Voting/CongressionalAssemble/index.js

```diff
@@ -163,62 +161,78 @@ function CongressionalAssemble() {
-      && <CurrentAssemble currentCongressMembers={democracy?.democracy?.currentCongressMembers} />}
-        {selectedCandidates && (
-        <CandidateVoting
-          handleUpdate={handleUpdate}
-          eligibleUnselectedCandidates={eligibleUnselectedCandidates}
-          selectedCandidates={selectedCandidates}
-          selectCandidate={selectCandidate}
-          unselectCandidate={unselectCandidate}
-          moveSelectedCandidate={moveSelectedCandidate}
-          didChangeSelectedCandidates={didChangeSelectedCandidates}
-        />
-        )}
-        {termDuration && (
-          <CongressionalCountdown termDuration={termDuration.toNumber()} />
-        )}
-      </div>
+      <Modal
+        open={isModalOpen}
+        title="Are you certain you want to leave the page?"
+        onOk={() => handleUpdate()}
+        onCancel={handleDiscardChanges}
+        okText="Update vote"
+        cancelText="Cancel and leave the page"
+      >
+        Your voting preferences haven&#96;t been saved, would you like to save them?
+      </Modal>
+      <Collapse
+        defaultActiveKey={['current', 'voting', 'preference', 'term']}
+        items={[
+          democracy?.democracy?.currentCongressMembers && {
+            key: 'current',
+            label: 'Acting Congressional Assembly',
+            children: (
+              <CurrentAssemble currentCongressMembers={democracy?.democracy?.currentCongressMembers} />
+            ),
+          },
+          selectedCandidates?.length && {
+            key: 'voting',
+            label: 'Voting',
+            children: (
+              <List
+                grid={{ gutter: 16 }}
+                dataSource={eligibleUnselectedCandidates}
+                renderItem={(eligibleUnselectedCandidate) => (
+                  <CandidateCard
+                    politician={eligibleUnselectedCandidate}
+                    selectCandidate={selectCandidate}
+                  />
+                )}
+              />
+            ),
+          },
+          {
+            key: 'preference',
+            label: 'My preference ordered Votes',
+            extra: (
+              <Button
+                primary
+                disabled={!didChangeSelectedCandidates}
+                onClick={() => handleUpdate()}
+              >
+                Update vote
+              </Button>
+            ),
+            children: selectedCandidates?.length ? (
+              <List
+                dataSource={selectedCandidates}
+                grid={{ gutter: 16 }}
+                renderItem={(currentCandidateVoteByUser) => (
+                  <SelectedCandidateCard
+                    politician={currentCandidateVoteByUser}
+                    unselectCandidate={unselectCandidate}
+                    moveSelectedCandidate={moveSelectedCandidate}
+                  />
+                )}
+              />
+            ) : <Alert type="info" message="No selected candidates" />,
```

Comment:
doesnt show candidates even when candidates exist.

Candidates are current members, runnerups plus candidates, all in one list. Idk where the bug is, but just for context

Reply:
Yeah I put in a the wrong list, fixed